worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -873,6 +851,56 @@ public IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set) {
         return null;
     }
 
+    private boolean NoBlacklistTrimXZ(
+            int lowerX,
+            int upperX,
+            int lowerZ,
+            int upperZ,
+            int upperZi,
+            int lowerZi,
+            boolean trimX,
+            boolean trimZ,
+            Object arr
+    ) {
+        if (!(trimX || trimZ)) {
+            return true;
+        }
+        int indexY = 0;
+        for (int y = 0; y < 16; y++, indexY += 256) { // For each y layer within a chunk section
+            int index;
+            if (trimZ) {
+                index = indexY;
+                for (int z = 0; z < lowerZ; z++) {
+                    // null the z values
+                    for (int x = 0; x < 16; x++, index++) {
+                        Array.set(arr, index, 0);
```

Comment:
This is far less performant than operating on the arrays directly. I'd rather the method be split into int vs char than have it "cleaner" tbh

Reply:
Fixed in latest commit!